.gitignore

```diff
@@ -1,2 +1,5 @@
 scalalib/src/
 target/
+
+*.dot
+*.ppm
```

Comment:
PPM files are not typical of the project and I don't think it's a good idea to have them in `.gitignore`. Ignoring `.dot` files is quite reasonable. Please drop the blank line and ppm and update your commit.